### ISSUE TITLE
Amazon Managed Grafana L2 Constructs

### ISSUE THREAD

**Issue by youngjeong46** [aws/aws-cdk-rfcs#471]
## Description

This is the RFC tracking issue for the creation of Amazon Managed Service for Grafana (AMG) workspaces using L2 Constructs. AMG only supports [L1 Constructs](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_grafana-readme.html).

## Roles

| Role                | User
|---------------------|------------------------------
| Proposed by         | @youngjeong46
| Author(s)           | @alias, @alias, @alias
| API Bar Raiser      | @rix0rrr 
| Stakeholders        | @alias, @alias, @alias

> See [RFC Process](https://github.com/aws/aws-cdk-rfcs#rfc-process) for details

## Workflow

- [x] Tracking issue created (label: `status/proposed`)
- [x] API bar raiser assigned (ping us at [#aws-cdk-rfcs](https://cdk-dev.slack.com/archives/C025ZFGMUCD) if needed)
- [ ] Kick off meeting
- [ ] RFC pull request submitted (label: `status/review`)
- [ ] Community reach out (via Slack and/or Twitter)
- [ ] API signed-off (label `api-approved` applied to pull request)
- [ ] Final comments period (label: `status/final-comments-period`)
- [ ] Approved and merged (label: `status/approved`)
- [ ] Execution plan submitted (label: `status/planning`)
- [ ] Plan approved and merged (label: `status/implementing`)
- [ ] Implementation complete (label: `status/done`)

---

> Author is responsible to progress the RFC according to this checklist, and
apply the relevant labels to this issue so that the RFC table in README gets
updated.

**Comment by rix0rrr** [comment]
👋 hi! I'll be your friendly neighbourhood bar raiser today 😀.

Don't know a lot about Grafana and how it will work, so please ELI5 the RFC...  but I did do the CloudWatch construct library and I know a bit about the dashboards there.

**Comment by mrgrain** [comment]
Marking this RFCs as `stale` since there has been little recent activity and it is not currently close to getting accepted as-is. We appreciate the effort that has gone into this proposal. Marking an RFCs as stale is not a one-way door. If you have made substantial changes to the proposal, please open a new issue/RFC. You might also consider raising a PR to [aws/aws-cdk](https://github.com/aws/aws-cdk/) directly or self-publishing to Construct Hub.
